netci.groovy

```diff
@@ -38,7 +38,7 @@ def project = 'dotnet/wcf'
             // so we set it first, and then call build.sh
             newCommitJob = job(Utilities.getFullJobName(project, jobName, false)) {
                 steps {
-                    shell("HOME=\$WORKSPACE/tempHome ./build.sh /p:OSGroup=${os} /p:Configuration=${os}_${configuration}")
+                    shell("HOME=\$WORKSPACE/tempHome ./build.sh /p:ShouldCreatePackage=false /p:OSGroup=${os} /p:Configuration=${os}_${configuration}")
```

Comment:
Incidentally, I was trying the /p:ShouldGenerateNuSpec parameter based on a hunch looking at the targets files, but didn't get a chance to try that change in CI.